### PR TITLE
MarketplaceOffer technical committee feedback

### PR DESCRIPTION
**Pull request by mthiers**
new Version of MarketplaceOffer with sounded feedback from technical committee

## Description
Attachment won't be changed due to semantic standards. Nevertheless, characteristics will be changed in the future to cover upload of multiple attachments -> DONE: changed to set already (we agreed to change it if there were already more changes to the model)
Quantity will be changed as in the comment above from Marcel. In addition the description will be changed to "The available quantity of an offered product". -> DONE
Status will be excluded as above -> DONE
Certificate will be excluded as above -> DONE
GeographicalCoordinate: Add description to: Location coordinates -> Description already included in longitude and latitude, therefore description of geographical coordinates given

Closes #

## MS2 Criteria
(to be filled out by PR reviewer)
- [ ] the model **validates** with the BAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar bamm-cli.jar -i \<path-to-aspect-model\> -v ). The  BAMM CLI is available [here](https://openmanufacturingplatform.github.io/sds-documentation/sds-developer-guide/dev-snapshot/tooling-guide/bamm-cli.html) and in [GitHub](https://github.com/OpenManufacturingPlatform/sds-sdk/releases)
- [ ] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [ ] the identifiers for all model elements **start with a capital letter** except for properties
- [ ] the identifier for **properties starts with a small letter**
- [ ] all model elements **at least contain the fields "name" and "description"** in English language. 
- [ ] **no duplicate names or preferredNames** within an Aspect (e.g. a Property and the referenced Characteristic should not have the same name)
- [ ] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [ ] use **abbreviations only when necessary** and if these are sufficiently common
- [ ] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [ ] fields `preferredName` and `description` are not the same
- [ ] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [ ] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [ ] units are referenced from the BAMM unit catalog whenever possible
- [ ] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [ ] when relying on **external standards**, they are referenced through a **"see"** element
- [ ] all properties with an [simple type](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/datatypes.html) have an example value
- [ ] metadata.json exists with status "release"

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [ ] All required reviewers have approved this PR (see reviewers section)
- [ ] The new aspect (version) will be implemented by at least one data provider
- [ ] The new aspect (version) will be consumed by at least one data consumer
- [ ] There exists valid test data
- [ ] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [ ] The model has at least version '1.0.0'

### CODE REVIEW

File: io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl [hunk 1]
```diff
@@ -0,0 +1,344 @@
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
```
Comment (bs-jokri) [inline io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl:1]: Add copyright and license header

File: io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+    bamm:exampleValue "1002090, xYz.09, ABcXYZ".
+:quantity a bamm:Property;
+    bamm:description "The available quantity of an offered product."@en;
+    bamm:characteristic :QuantityCharacteristic;
+    bamm:exampleValue "50"^^xsd:double.
+:unitOfMeasure a bamm:Property;
+    bamm:description "The unit of measure (related to quantity)."@en;
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "each";
+    bamm:see <file:///C:/Program%20Files/Bosch_BAMM/https%253A%252F%252Fresources.gs1us.org%252FGS1-US-Data-Hub-Help-Center%252FArtMID%252F3451%252FArticleID%252F116%252FUnit-of-Measure-Codes>.
```
Comment (bs-jokri) [inline io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl:70]: Remove local path

File: io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+    bamm:characteristic :ConditionCharacteristic;
+    bamm:exampleValue "used".
+:pickupLocation a bamm:Property;
+    bamm:description "The pickup location for the offered item."@en;
+    bamm:characteristic :GeographicalCoordinates.
+:incoterms a bamm:Property;
+    bamm:description "The incoterms."@en;
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "DAP (Delivered at Place)";
+    bamm:see <file:///C:/Program%20Files/Bosch_BAMM/https%2525253A%2525252F%2525252Ficcwbo.org%2525252Fpublication%2525252Fincoterms-2020-introduction%2525252F>.
```
Comment (bs-jokri) [inline io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl:82]: remove local path

File: io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+    bamm:description "The price of the product."@en;
+    bamm:characteristic :Price.
+:image a bamm:Property;
+    bamm:description "Images of the product."@en;
+    bamm:characteristic :ImageFile.
+:attachment a bamm:Property;
+    bamm:description "Attachments related to the product."@en;
+    bamm:characteristic :Attachment.
+:bundleOffer a bamm:Property;
+    bamm:preferredName "bundle Offer"@en;
```
Comment (bs-jokri) [inline io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl:93]: Bundle Offer or bundle offer

File: io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+    bamm:characteristic :CatenaXIdTrait;
+    bamm:exampleValue "580d3adf-1981-44a0-a214-13d6ceed9379".
+:ConditionCharacteristic a bamm-c:Enumeration;
+    bamm:preferredName "Condition Characteristic"@en;
+    bamm:description "Condition of the article as enumeration."@en;
+    bamm:dataType xsd:string;
+    bamm-c:values ("New" "Used" "Refurbished" "Other").
+:GeographicalCoordinates a bamm:Characteristic;
+    bamm:preferredName "geographical coordinates"@en;
+    bamm:see <file:///C:/Program%20Files/Bosch_BAMM/https%2525253A%2525252F%2525252Fen.wikipedia.org%2525252Fwiki%2525252FGeographic_coordinate_system>;
```
Comment (bs-jokri) [inline io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl:152]: remove local path

File: io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+    bamm:properties (:longitude :latitude).
+:PriceEntity a bamm:Entity;
+    bamm:preferredName "price"@en;
+    bamm:properties (:value :currency).
+:File a bamm:Entity;
+    bamm:preferredName "File"@en;
+    bamm:description "Image file of the object."@en;
+    bamm:properties (:name :location).
+:AttachmentEntity a bamm:Entity;
+    bamm:preferredName "AttachmentEntity"@en;
```
Comment (bs-jokri) [inline io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl:199]: attachment entity

File: io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+:value a bamm:Property;
+    bamm:preferredName "value"@en;
+    bamm:description "The absolute value of the price"@en;
+    bamm:characteristic :Value;
+    bamm:exampleValue "250.00"^^xsd:float.
+:currency a bamm:Property;
+    bamm:description "The currency of the price."@en;
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "EUR";
+    bamm:see <file:///C:/Program%20Files/Bosch_BAMM/https%2525253A%2525252F%2525252Fwww.iso.org%2525252Fiso-4217-currency-codes.html>.
```
Comment (bs-jokri) [inline io.catenax.market_place_offer/1.4.0/MarketplaceOffer.ttl:244]: remove local path

**Review by bs-jokri — CHANGES_REQUESTED** [review]
check comments

**Comment by mthiers** [comment]
@bs-jokri 

as agreed in todays core team modelling round: after MS2 is provided MS3 from product is given, as long as models are unchaged (shared meeting between technical committee and product with discussion iof changes)

**Comment by bs-jokri** [comment]
ECA check is not successful.
https://accounts.eclipse.org/legal/eca/validation/138411

@mthiers please check the email address used.

**Comment by bs-jokri** [comment]
Superseded by #67

**Pull request closed without merging**